README.md

```diff
@@ -1,2 +1,66 @@
 # web-components-starter
-Starter React app for using AP web components
+Starter React app for using AP web components.
+
+To develop:
+
+```
+npm i
+npm start
+```
+
+To build:
+
+```
+npm build
+```
+---
+
+<a href="https://www.accordproject.org/">
+  <img src="assets/APLogo.png" alt="Accord Project Logo" width="400" />
```

Comment:
Will want to change this to https://github.com/accordproject/techdocs/blob/master/assets/APLogo.png

Reply:
Or add the asset.

Reply:
good catch!